### PR TITLE
adding secondary for Run3Summer22DRPremix and Run3Summer22DR

### PR DESCRIPTION
**Pull request by tyjyang**
@haozturk @z4027163 Per request from [this](https://its.cern.ch/jira/browse/PRCAMPAIGNS-298) and [this](https://its.cern.ch/jira/browse/PRCAMPAIGNS-297) ticket

### CODE REVIEW

**Comment by haozturk** [comment]
Thanks Justin: Couple of comments:
1. Format of the secondaries should be as 
```
    "secondaries": {
      "/MinBias_TuneCP5_14TeV-pythia8/Run3Winter20GS-110X_mcRun3_2021_realistic_v6-v1/GEN-SIM": {
        "SiteWhitelist": [
          "T1_UK_RAL_Disk",
          "T1_US_FNAL_Disk"
        ]
      }
    },
```
for classical mixing pileups
and 
```
    "secondaries": {
      "/Neutrino_E-10_gun/RunIISummer20ULPrePremix-UL16_106X_mcRun2_asymptotic_v13-v1/PREMIX": {
        "SecondaryLocation": [
          "T1_US_FNAL_Disk", 
          "T2_CH_CERN"
        ]
      }
    }, 
```
for PREMIX pileups.

We should always specify and enforce locations for secondaries. P&R controls the pileup locations. Are these pileups already produced and ready? If so, we should decide their locations

2. Specify secondary AAA
```
"secondary_AAA": true, 
```
it should be `true` for PREMIX and `false` for classical mixing.

**Comment by tyjyang** [comment]
@haozturk Thanks for the information. I have checked the locations of the pileup sets and changed the configs accordingly.

File: campaigns.json [hunk 1]
```diff
@@ -1012,11 +1012,29 @@
...
+      },
+      "secondary AAA": false
   },
   "Run3Summer22DRPremix": {
       "fractionpass": 0.95,
-      "go": false
+      "go": false,
+      "secondaries": {
+        "/Neutrino_E-10_gun/Run3Summer21PrePremix-Summer22_124X_mcRun3_2022_realistic_v11-v2/PREMIX": {
+          "SiteWhitelist": [
```
Comment (haozturk) [inline campaigns.json:1031]: You should use `SecondaryLocation` location for Neutrinos. (I know it's very confusing, this is planned to be improved)

**Comment by tyjyang** [comment]
@haozturk changed it, thanks for the correction

**Comment by haozturk** [comment]
Thanks a lot Justin, things look good

**Pull request merged**